analysis/02_global_clustering/global_clustering.Rmd

```diff
@@ -86,7 +83,7 @@ hist(promoter_positive_peaks)
 
```

Comment:
Looks there's still a bit to clean up here, for ex. line 79 still has HINT. We'll wait to merge it until all this is cleaned up.